editor/script_create_dialog.cpp

```diff
@@ -209,6 +193,67 @@ void ScriptCreateDialog::set_inheritance_base_type(const String &p_base) {
 	base_type = p_base;
 }
 
+void ScriptCreateDialog::set_languages_list(const bool p_only_attachable) {
+	only_attachable = p_only_attachable;
```

Comment:
You are calling this method after creating dialog, before it enters tree, so the list will be initialized twice (the second time in NOTIFICATION_THEME_CHANGED). You can add early return if the node is not inside tree yet.